src/hooks/useGovernance.ts

```diff
@@ -0,0 +1,102 @@
+import { computed, ref, onMounted } from 'vue';
+import { useNetworkInfo } from './useNetworkInfo';
+import axios from 'axios';
+
+export type GovernanceData = {
+  title: string;
+  index: number;
+  state: string;
+  url: string;
+};
+
+const proposals = ref<GovernanceData[]>([]);
+const ongoingReferenda = ref<GovernanceData>();
+
+const fetchProposals = async (network: string): Promise<GovernanceData[]> => {
+  try {
+    const url = `https://${network}.subsquare.io/api/democracy/proposals?simple=true&page=1&page_size=2`;
```

Comment:
is this page size is because we need to fetch only two proposals?

Reply:
Yes, it is because of that